server/routes/recipeRoutes.js

```diff
@@ -31,6 +32,9 @@ router.post('/addRecipe', authenticate, addRecipe);
 // delete recipe
 router.delete('/deleteRecipe/:id', authenticate, deleteRecipe);
 
+// remove all recipes
+router.delete('/deleteAllRecipes', removeAllRecipes);
```

Comment:
I was actually joking when I suggested this idea lol. Though what you can do though is leave the code you have in the recipeUtils file and just call that function directly in the tests where you need it

Reply:
For some reason, when I tried to call the function in recipeUtils directly, the compiler flagged me with `Error: Dialect needs to be explicitly supplied as of v4.0.0`. Do you happen to see this issue before? If this is not worth the trouble, I think we can just leave it like this.

Reply:
Ah I think I know the problem is. I got that error when I was providing a bad URI to connect to the DB with. First try to copy the DB_URI value from the .env file in the server directory to the one in the top level directory. If that doesn't work, try adding this code at the top of your test file. 
```javascript
const { Sequelize } = require('sequelize');
const sequelize = new Sequelize(process.env.DB_URI);
```
This should form the connection to the db and resolve the error

Reply:
The fix you recommended work, but then I start running into timeout issue. I tried going to the function in recipeUtils and add a timeout of 5000 ms, and it doesn't seem to work. Still not sure how to tackle this.